### PR TITLE
Datadog setup for broker

### PR DESCRIPTION
**Pull request by hsharma96**

### CODE REVIEW

File: dataactbroker/app.py [hunk 1]
```diff
@@ -21,6 +21,13 @@
 from dataactcore.utils.responseException import ResponseException
 from dataactcore.utils.statusCode import StatusCode
 
+# Datadog import
+USE_DATADOG = False
```
Comment (michaeldhess) [inline dataactbroker/app.py]: Add a comment here to let developers know that this value is being changed with Ansible, so nobody deletes it thinking the next `if` can never be reached.

File: dataactvalidator/app.py [hunk 2]
```diff
@@ -17,6 +17,13 @@
 from dataactvalidator.validation_handlers.validationError import ValidationError
 from dataactvalidator.validation_handlers.validationManager import ValidationManager
 
+# Datadog import
+USE_DATADOG = False
```
Comment (michaeldhess) [inline dataactvalidator/app.py]: Add a comment here to let developers know that this value is being changed with Ansible, so nobody deletes it thinking the next `if` can never be reached.

File: dataactvalidator/health_check.py [hunk 3]
```diff
@@ -4,6 +4,13 @@
 from dataactcore.logging import configure_logging
 from dataactcore.utils.jsonResponse import JsonResponse
 
+# Datadog import
+USE_DATADOG = False
```
Comment (michaeldhess) [inline dataactvalidator/health_check.py]: Add a comment here to let developers know that this value is being changed with Ansible, so nobody deletes it thinking the next `if` can never be reached.

File: dataactvalidator/health_check.py [hunk 4]
```diff
@@ -27,6 +34,11 @@ def test_app():
 def run_app():
     """Run the application."""
     flask_app = create_app()
+
+# This is for DataDog
```
Comment (michaeldhess) [inline dataactvalidator/health_check.py]: This comment should be tabbed over.

File: dataactvalidator/app.py [hunk 5]
```diff
@@ -28,6 +35,10 @@ def run_app():
     """Run the application."""
     app = create_app()
 
+# This is for DataDog
```
Comment (michaeldhess) [inline dataactvalidator/app.py]: This comment should be tabbed over.

File: dataactbroker/app.py [hunk 6]
```diff
@@ -98,6 +105,11 @@ def handle_exception(exception):
 def run_app():
     """runs the application"""
     flask_app = create_app()
+
+# This is for DataDog
```
Comment (michaeldhess) [inline dataactbroker/app.py]: This comment should be tabbed over.

**Comment by hsharma96** [comment]
@michaeldhess added comments and fixed spacing

**Pull request merged**